cves/kernel/CVE-2018-12232.yml

```diff
@@ -311,9 +324,12 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: true
+  note: |
```

Comment:
Comma-separate these

Reply:
Good idea! I just put them on new lines to indicate separate links but I'll add the commas.